src/main/java/executor/service/config/properties/PropertiesConstants.java

```diff
@@ -15,6 +15,16 @@ private PropertiesConstants() {
     public static final String THREAD_POOL_PROPERTIES = "thread-pool.properties";
     public static final String CORE_POOL_SIZE = "core.pool.size";
     public static final String KEEP_ALIVE_TIME = "keep.alive.time";
```

Comment:
Why only a small part of the constants is being used? Maybe in such case this class should be removed at all?

Reply:
We directly speak about this class with Olexander, but he insists that this class is necessary...